src/languages/opengoal/disasm/opengoal-disasm-renamer.ts

```diff
@@ -15,7 +15,10 @@ export class OpenGOALDisasmRenameProvider implements vscode.RenameProvider {
     newName: string,
     token: vscode.CancellationToken
   ): Promise<vscode.WorkspaceEdit | undefined> {
-    const symbolRange = document.getWordRangeAtPosition(position, /[\w\-.]+/g);
+    const symbolRange = document.getWordRangeAtPosition(
+      position,
+      /[\w\-.!?+/*%]+/g
+    );
```

Comment:
it's probably better to define what _isn't_ allowed in a symbol than what is, else this will be a thing that needs constant updating.

I'd suggest trying the following regex instead `/[^\s\(\)]+/`